src/main/java/org/nuxeo/onedrive/client/OneDriveFolder.java

```diff
@@ -104,6 +110,28 @@ public static OneDriveFolder getRoot(OneDriveAPI api) {
         return () -> new OneDriveItemIterator(getApi(), url);
     }
 
+    public OneDriveItemIterator delta() {
+        return delta(null);
+    }
+
+    public OneDriveItemIterator delta(String deltaLink) {
+        URL url;
+            if (deltaLink!=null) {
```

Comment:
I would rather not having this condition here, by leaving the code using deltaLink here, and the other part directly in delta().